server/views.py

```diff
@@ -102,6 +103,9 @@ def show_all_volunteers(request, page=1):
     if len(guidings) != 0:
         filter_options['guiding'] = True
 
+    if len(city_name) != 0 and '' not in city_name:
```

Comment:
Isn't ('' not in city_name) always false? When will an empty string be in a string?

Reply:
That's because we get a list from GET, and not a string. Until total refactoring, this is the standard (to keep uniformity)